src/features/autodelete-spam.ts

```diff
@@ -20,6 +21,37 @@ const safeDomains = [
 const checkWords = (message: string, wordList: string[]) =>
   message.split(/\b/).some((word) => wordList.includes(word.toLowerCase()));
 
+const getPingCount = (content: string) => {
```

Comment:
I wanted to use [`message.pings.everyone`](https://discord.js.org/#/docs/discord.js/stable/class/MessageMentions?scrollTo=everyone) but it isn't true if the ping didn't work, apparently. Have to parse them out of the text